src/util/loanUtils.js

```diff
@@ -75,6 +75,22 @@ export function buildPriceArray(amountLeft, minAmount) {
 	return priceArray;
 }
 
+export function build5DollarsPriceArray(amountLeft) {
+	const limit5 = amountLeft < 50 ? amountLeft : 50;
+	const N_5 = limit5 / 5;
```

Comment:
It might be helpful for these variables to be more descriptive. I'm assuming this is number of $5 options? Something like `numberOf5` would be more clear when glancing through the code.